stt_main.py

```diff
@@ -334,11 +283,11 @@ def menu_pick(self):
         if mainmenu == '1':
             assessment_id = str(input("Please choose one of the assessments; \n1. Five Points \n2. Random N \n"))
             if assessment_id == '1':
-                self.assessment_FivePoint()
+                ass.assessment_FivePoint()
                 self._assessment = 'FivePoint'
 
             if assessment_id == '2':
```

Comment:
use elif. also add else statement for error analysis
`else: print("input number between 1-2)`